easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-2.0.0-intel-2018a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'BerkeleyGW'
+version = '2.0.0'
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+ properties and the optical responses of a large variety of materials from bulk periodic crystals to
+ nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://berkeley.box.com/shared/static/']
+sources = [{'download_filename': 'wkqu37wu77tll53r8t1soe6ozqja67yy.gz', 'filename': SOURCE_TAR_GZ}]
```

Comment:
@migueldiascosta this is the best option? :-/

Reply:
@boegel afaik, it's the only option... :(

Reply:
Maybe we should reach out to them and suggest to properly name their tarballs?

Mentioning `BGW-2.0.0.tar.gz` on the download page and then downloading a tarball that's named like gibberish doesn't make any sense...